lineman/app/components/thread_preview_collection/thread_preview_collection.scss

```diff
@@ -1,4 +1,4 @@
-.thread-previews-container {
+.inbox-page__groups .thread-previews-container, .dashboard-page__collections .thread-previews-container {
```

Comment:
This is almost great. I'm wondering why there are qualifiers here. I mean, are there thread-previews-containers which won't work with this css? if so, why? If not.. then we should be able to drop the qualifiers here and just use .thread-previews-container as the selector